pkg/operator/hive/hive.go

```diff
@@ -70,12 +71,13 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 		)
 	}
 
+	disabledControllersFlag := assembleDisabledControllersFlag(instance.Spec.DisabledControllers)
+	if disabledControllersFlag != "" {
+		hiveContainer.Args = append(hiveContainer.Args, disabledControllersFlag)
+	}
```

Comment:
Does this need a specific function?
```suggestion
	if dc := instance.Spec.DisabledControllers; len(dc) != 0 {
		hiveContainer.Args = append(hiveContainer.Args, "--disabled-controllers", strings.Join(dc, ","))
	}
```

Reply:
Nope. It was more complex when I took a first pass at it, and then the function just stuck around. I'll condense it down.